app/assets/stylesheets/categories.scss

```diff
@@ -1,3 +1,73 @@
 // Place all the styles related to the categories controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+
+.categories {
+  position: relative;
+}
+
+.scan {
+  // position: absolute;
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.